src/Nethermind/Nethermind.Init/Steps/InitializeBlockchain.cs

```diff
@@ -84,8 +84,6 @@ private Task InitBlockchain()
                 syncConfig.DownloadBodiesInFastSync = true;
             }
 
-            Account.AccountStartNonce = getApi.ChainSpec.Parameters.AccountStartNonce;
```

Comment:
Why is this removed? Might be important.

Reply:
A remanent from a different age
```
        /// <summary>
        /// This is a special field that was used by some of the testnets (namely - Morden and Mordor).
        /// It makes all the account nonces start from a different number then zero,
        /// hence preventing potential signature reuse.
        /// It is no longer needed since the replay attack protection on chain ID is used now.
        /// We can remove it now but then we also need to remove any historical Mordor / Morden tests.
        /// </summary>
```